src/extension-support/extension-manager.js

```diff
@@ -133,6 +133,21 @@ class ExtensionManager {
         });
     }
 
+    /**
+    * regenerate blockinfo for any loaded extensions
+    */
+    refreshBlocks () {
+        this._loadedExtensions.forEach(serviceName => {
```

Comment:
This function is so tiny now! 🐭

Reply:
💯  yes I like how simple this one ends up!